test/resources/openshift_auth.go

```diff
@@ -137,7 +137,7 @@ func openshiftClientSetup(url, username, password string, client *http.Client, i
 	//sometimes we'll reach an accept permissions page for the user if they haven't accepted these scope requests before.
 	//refactored, this approach assumes that if the redirected page is not the console then it looks for an approve action, previous approach would cause e2e test flakes
 	if strings.Contains(browser.Url().Host, openshiftConsoleSubdomain) {
-		l.Log("Looking for permissions form on page", browser.Url().Host, browser.Body())
+		// l.Log("Looking for permissions form on page", browser.Url().Host, browser.Body())
```

Comment:
Should the changes in this file be reverted before merging?

Reply:
I'll revert it back, I just commented it out to make it easier to investigate the reason e2e test was not passing in PROW